sonic-slave-stretch/Dockerfile.j2

```diff
@@ -291,7 +291,16 @@ RUN apt-get update && apt-get install -y \
         automake1.11 \
         libselinux1-dev \
 # For kdump-tools
-        liblzo2-dev
+        liblzo2-dev \
+# For SAI
+        protobuf-compiler \
+        libprotobuf-dev \
+        pkg-config \
+        xxd
```

Comment:
add to depends in the libsai control file.

Reply:
I have removed all the packages **except** "libprotobuf-dev" which is a dependency for the libsaibcm module. I have to keep this package alone, as the current rules in the Dockerfile don't install dependency packages.